benchmark-cuqi-ct image error on launch

There is an issue with the current image, maybe the benchmark needs to be updated for a new cuqipy package version?

@jakobsj  @amal-ghamdi  @nabriis  Could you have a look at that?

`sudo docker run -p 4243:4243  linusseelinger/benchmark-cuqi-ct:latest`
```
Traceback (most recent call last):
  File "/server/server.py", line 10, in <module>
    from cuqi.distribution import Gaussian, GMRF, Laplace_diff, Cauchy_diff
ImportError: cannot import name 'Laplace_diff' from 'cuqi.distribution' (/usr/local/lib/python3.10/dist-packages/cuqi/distribution/__init__.py)
```

Now getting an error in test_output:
`   File "/home/runner/work/benchmarks/benchmarks/benchmarks/deconvolution-1d/test_output.py", line 85, in <module>
    TP.prior = cuqi.distribution.Cauchy_diff(np.zeros(128), 0.01)
AttributeError: module 'cuqi.distribution' has no attribute 'Cauchy_diff'
Error: Process completed with exit code 1.`
Sorry for missing that one earlier, the generic protocol check won't catch that of course :D

--- a/benchmarks/cuqi-ct/test_output.py
+++ b/benchmarks/cuqi-ct/test_output.py
@@ -12,7 +12,7 @@ from scipy.io import loadmat
 from cuqi.model import LinearModel
 from cuqi.geometry import Image2D
 from cuqi.array import CUQIarray
-from cuqi.distribution import Gaussian, GMRF, Laplace_diff, Cauchy_diff
+from cuqi.distribution import Gaussian, GMRF, LMRF, CMRF
 from cuqi.problem import BayesianProblem
 
 
@@ -136,14 +136,12 @@ BP.prior = cuqi.distribution.GMRF(np.zeros(256**2), 1/(0.01),
                                   geometry=BP.likelihood.geometry)
 assert output_GMRF == pytest.approx(BP.posterior.logpdf(parameters))
 
-BP.prior = cuqi.distribution.Cauchy_diff(np.zeros(256**2), 0.01,
-                                         physical_dim=2,
-                                         geometry=BP.likelihood.geometry)
+BP.prior = cuqi.distribution.CMRF(np.zeros(256**2), 0.01,
+                                  geometry=BP.likelihood.geometry)
 assert output_CMRF == pytest.approx(BP.posterior.logpdf(parameters))
 
-BP.prior = cuqi.distribution.Laplace_diff(np.zeros(256**2), 0.1,
-                                          physical_dim=2,
-                                          geometry=BP.likelihood.geometry)
+BP.prior = cuqi.distribution.LMRF(np.zeros(256**2), 0.1,
+                                  geometry=BP.likelihood.geometry)
 assert output_LMRF == pytest.approx(BP.posterior.logpdf(parameters))
 
 assert np.allclose(output_exactSolution, parameters)

--- a/benchmarks/deconvolution-1d/README.md
+++ b/benchmarks/deconvolution-1d/README.md
@@ -192,8 +192,8 @@ if prior == "Gaussian":
 elif prior == "GMRF":
     TP.prior = cuqi.distribution.GMRF(np.zeros(128), 1/delta)
 elif prior == "CMRF":
-    TP.prior = cuqi.distribution.Cauchy_diff(np.zeros(128), delta)
+    TP.prior = cuqi.distribution.CMRF(np.zeros(128), delta)
 elif prior == "LMRF":
-    TP.prior = cuqi.distribution.Laplace_diff(np.zeros(128), delta)
+    TP.prior = cuqi.distribution.LMRF(np.zeros(128), delta)
 TP.sample_posterior(2000).plot_ci(exact=TP.exactSolution)
 ```

--- a/benchmarks/deconvolution-1d/test_output.py
+++ b/benchmarks/deconvolution-1d/test_output.py
@@ -82,10 +82,10 @@ assert output_Gaussian == pytest.approx(TP.posterior.logpdf(parameters))
 TP.prior = cuqi.distribution.GMRF(np.zeros(128), 1/(0.01))
 assert output_GMRF == pytest.approx(TP.posterior.logpdf(parameters))
 
-TP.prior = cuqi.distribution.Cauchy_diff(np.zeros(128), 0.01)
+TP.prior = cuqi.distribution.CMRF(np.zeros(128), 0.01)
 assert output_CMRF == pytest.approx(TP.posterior.logpdf(parameters))
 
-TP.prior = cuqi.distribution.Laplace_diff(np.zeros(128), 0.01)
+TP.prior = cuqi.distribution.LMRF(np.zeros(128), 0.01)
 assert output_LMRF == pytest.approx(TP.posterior.logpdf(parameters))
 
 assert np.allclose(output_exactSolution, TP.exactSolution)

